src/main/java/com/metamx/common/StringUtils.java

```diff
@@ -84,4 +84,24 @@ public static String safeFormat(String message, Object... formatArgs)
       return bob.toString();
     }
   }
+
+  // should be used only for estimation.. does not check validity of format
```

Comment:
Basically the same but it does not validate whether the string has valid UTF format. For invalid chars, this method over-estimates the length(length of invalid char = 1 for encoding).